CHANGELOG.md

```diff
@@ -15,7 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix S3RDDLayerReader partitioning [#3231](https://github.com/locationtech/geotrellis/pull/3231)
 - GDALRasterSource works inconsistenly with BitCellType and ByteCellType [#3232](https://github.com/locationtech/geotrellis/issues/3232)
 - rasterizeWithValue accepts only topologically valid polygons [#3236](https://github.com/locationtech/geotrellis/pull/3236)
-- Rasterizer.rasterize should be consistent with rasterizeWithValue [#3238](https://github.com/locationtech/geotrellis/pull/3238) 
+- Rasterizer.rasterize should be consistent with rasterizeWithValue [#3238](https://github.com/locationtech/geotrellis/pull/3238)
```

Comment:
These two changes read like test assertions. Should we add tests to cover them?

Reply:
We can add an empty test [here](https://github.com/locationtech/geotrellis/blob/f1335c31e5408ab2bdae4cb3cd56343e459f752f/spark/src/test/scala/geotrellis/spark/stitch/CollectionStitchMethodsSpec.scala), agree.